app/decorators/person_decorator.rb

```diff
@@ -26,6 +26,12 @@ def project_with_contributions(project)
     person.project_with_contributions(project).map(&:name)
   end
 
+  def saying
+    given :saying do
+      content_tag(:blockquote, raw(person.saying), class: 'saying')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.